quinn-proto/src/connection/mod.rs

```diff
@@ -456,7 +456,7 @@ impl Connection {
             true => max_datagrams.min(MAX_TRANSMIT_SEGMENTS),
         };
 
-        let mut num_datagrams = 0;
+        let mut num_datagrams = 0usize;
```

Comment:
This and related changes don't seem to have anything to do with the intent of the commit. Please keep cosmetic refactoring separate.